returnn/tf/engine.py

```diff
@@ -877,20 +877,17 @@ def __init__(self, config=None):
         self.tf_session = None  # type: typing.Optional[tf.compat.v1.Session]
         self.network = None  # type: typing.Optional[TFNetwork]
         self.updater = None  # type: typing.Optional[Updater]
-        self.learning_rate_control = None  # type: typing.Optional[LearningRateControl]
         self._checked_uninitialized_vars = False
         self._merge_all_summaries = None
         self.dataset_batches = {}  # type: typing.Dict[str,BatchSetGenerator]
         self.dataset_provider = None  # type: typing.Optional[DatasetDataProvider]
         self.train_data = None  # type: typing.Optional[Dataset]
-        self.eval_datasets = {}  # type: typing.Dict[str,Dataset]
```

Comment:
Why did you remove this?

Reply:
I went a little ahead and also checked for unification of datasets, and this was a leftover I missed when doing the PR.